### PR TITLE
Added a bash script to perform memory profiling

### PR DESCRIPTION
**Pull request by czardoz**

### CODE REVIEW

File: npm/memory-check.sh [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+	local TEST_SCRIPT="memtest.js";
+
+	# Enter the version directory
+	cd ${VERSION_DIR};
+
+	git checkout ${VERSION} &> /dev/null;
+
+	# Create the test script
+	cat > ${VERSION_DIR}/${TEST_SCRIPT} <<-EOF
+	var sdk = require('postman-collection'),
```
Comment (kunagpal) [inline npm/memory-check.sh]: The code here could be take from a test template.

Comment (kunagpal) [inline npm/memory-check.sh]: Also, an exception for this script will have to be added to the package script system tests.

Comment (czardoz) [inline npm/memory-check.sh]: Better to leave the script self contained. Any errors will cause the memory tests to stop anyway.

Comment (czardoz) [inline npm/memory-check.sh]: But we can do this if you are hung up on it :+1:

Comment (czardoz) [inline npm/memory-check.sh]: Oh no, we can't.. need to be able to profile older versions too.

**Pull request merged**